app/models/user.rb

```diff
@@ -41,6 +45,18 @@ def mailchimp_status
     end
   end
 
+  def versions
+    PaperTrail::Version.where(whodunnit: id)
```

Comment:
This is a code smell - why does the user need to know to look for a version of a case, or how that process works? Keep in mind that at some point, we may want to add versioning to other entities like agencies, organizations, or events - the way it's written right now that won't work.

I like the idea of the user noting the last time they edited a case, but not like this.

Reply:
This method doesn't actually get used at this point. I was thinking of how to move the new controller instance variables into model scopes and thought this would be helpful for that purpose, but decided against it. Then I was thinking it would eventually be useful to show on the user profile page what cases a person has edited. I will just remove it though.